specification/window.browser.bs

```diff
@@ -52,6 +52,38 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     it MUST be used only for WebExtension related functionality.
   </section>
 
+  <section>
+    For backwards compatibility with older versions of WebExtensions that used
+    <code>chrome</code> as a global identifier for WebExtension related methods,
+    a UA MAY define a global <code>chrome</code> object that includes all
+    properties of {{browser}}.
+    {{browser}} and <code>chrome</code> MAY be direct aliases of each other.
+    When {{browser}} and <code>chrome</code> are distinct objects, each initial
+    property of {{browser}} SHOULD be defined on the <code>chrome</code> object,
+    and each property descriptor on the <code>chrome</code> object SHOULD return
+    the same value as the corresponding property on the {{browser}} object.
+
+    <div class="example" id="example-1">
+      Here's an example of the expected alias behavior
+      <pre><code class="lang-js">
+        // As `chrome` and `browser` are not direct aliases,
+        // modifying one top level values will not change the other
```

Comment:
```suggestion
        // When `chrome` and `browser` are not direct aliases,
        // modifying one top level values will not change the other:
```

Rephrased to avoid the impression that chrome and browser couldn't be direct aliases.